queries/php/injections.scm

```
@@ -1,3 +1,3 @@
 (text) @html
 
-(comment) @comment
+(comment) @phpdoc
```

I'm not sure if this can be a problem, as this parser is marked as experimental. So what happens if a PHP user has not installed the phpdoc parser? Will this be silently ignored?

---

Tested locally (`:TSUninstall phpdoc`). If phpdoc is not installed comment blocks are simply not parsed which is fine IMO.

---

yes, injections are ignored when the parser is not installed.